omero/users/whatsnew.rst

```diff
@@ -4,7 +4,7 @@ What's new for OMERO 5.4 for users
 Updates and new features for OMERO 5.4 include:
 
 - Your system administrator can now create
-  :doc:`/sysadmins/admins-with-restricted-privileges` to allow facility
+  :doc:`/sysadmins/restricted-admins` to allow facility
```

Comment:
As discussed, add the link to the whatsnew for users from the users index page @hflynn

Reply:
Pushed now @pwalczysko is that location clear enough for do you want me to add it to a bullet list instead?

Reply:
Relisted for tomorrow with my name. Will have one more look at the commits pushed today